gslib/tests/test_gcs_json_credentials.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for gcs_json_credentials.py."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+from apitools.base.py import GceAssertionCredentials
+from google_reauth import reauth_creds
+from gslib import gcs_json_api
+from gslib import gcs_json_credentials
+from gslib.cred_types import CredTypes
+from gslib.exception import CommandException
+from gslib.tests import testcase
+from gslib.tests.util import SetBotoConfigForTest
+from gslib.utils.wrapped_credentials import WrappedCredentials
+import logging
```

Comment:
std lib imports usually go on  the top. go/pystyle#imports-formatting

Reply:
Done